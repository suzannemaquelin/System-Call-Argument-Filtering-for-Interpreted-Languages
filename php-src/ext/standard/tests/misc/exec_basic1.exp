Warning: exec(): NULL byte detected. Possible attack in %s on line %d
bool(false)
NULL

Warning: system(): NULL byte detected. Possible attack in %s on line %d
bool(false)

Warning: passthru(): NULL byte detected. Possible attack in %s on line %d
bool(false)