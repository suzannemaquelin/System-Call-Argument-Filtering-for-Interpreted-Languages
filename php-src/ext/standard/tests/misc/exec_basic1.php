<?php
$cmd = "echo abc\n\0command";
var_dump(exec($cmd, $output));
var_dump($output);
var_dump(system($cmd));
var_dump(passthru($cmd));
?>
