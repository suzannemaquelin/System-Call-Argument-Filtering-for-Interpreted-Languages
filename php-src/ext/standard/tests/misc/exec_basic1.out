Failed loading /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so:  /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so: cannot open shared object file: No such file or directory

Warning: exec(): NULL byte detected. Possible attack in /root/php-src/ext/standard/tests/misc/exec_basic1.php on line 3
bool(false)
NULL

Warning: system(): NULL byte detected. Possible attack in /root/php-src/ext/standard/tests/misc/exec_basic1.php on line 5
bool(false)

Warning: passthru(): NULL byte detected. Possible attack in /root/php-src/ext/standard/tests/misc/exec_basic1.php on line 6
bool(false)